telemetry/definitions/commonDefinitions.json

```
@@ -239,6 +239,16 @@
             "name": "ecrDeploySource",
             "allowedValues": ["dockerfile", "tag"],
             "description": "The source content specified in the ECR deployment request"
+        },
+        {
+            "name": "dydbFetchType",
+            "allowedValues": ["scan", "query"],
+            "description": "The type of fetch being performed"
+        },
+        {
+            "name": "dydbIndexType",
```

if we can't generalize this to something service-agnostic, I would suggest at least we don't introduce different service prefixes (dydb vs dynamodb)
```suggestion
            "name": "dynamodbIndexType",
```

it is unfortunate that we didn't use the `{prefix}_{name}` (with underscore) convention as in the metric names. Probably shouldn't start using underscores in type names I guess.